GDSCommon-Demo/GDSCommon-DemoTests/GDSListOptionsViewControllerTests.swift

```diff
@@ -104,30 +120,63 @@ extension ListOptionsViewControllerTests {
         try sut.primaryButton.sendActions(for: .touchUpInside)
         XCTAssertEqual(didSetStringKey, "Table view list item 1")
     }
+    
+    func testSecondaryButton() throws {
+        setupSUTWithOptionals()
+        XCTAssertFalse(try sut.secondaryButton.isHidden)
+        
+        XCTAssertFalse(didDismiss)
+        try sut.secondaryButton.sendActions(for: .touchUpInside)
+        XCTAssertTrue(didDismiss)
+    }
+    
+    func testContentHiddenWhenNil() throws {
+        XCTAssertTrue(try sut.secondaryButton.isHidden)
+        XCTAssertTrue(try sut.stackView.isHidden)
+        XCTAssertNil(try sut.tableTitleLable.text)
+    }
```

Comment:
Perhaps a testContentNotHiddenWhenNotNil, or similar, as that would have caught the error I flagged above?